src/gui/tray/UserModel.h

```diff
@@ -75,6 +75,7 @@ public slots:
     void slotItemCompleted(const QString &folder, const SyncFileItemPtr &item);
     void slotProgressInfo(const QString &folder, const ProgressInfo &progress);
     void slotAddError(const QString &folderAlias, const QString &message, ErrorCategory category);
+    void slotAddErrorToGui(const QString &folderAlias, SyncFileItem::Status status, const QString &errorMessage, const QString &subject = {});
```

Comment:
what is the purpose of the fourth arg ? you are never adding it so it is always empty

Reply:
@mgallien One can pass a subject that gets displayed with the error message in GUI. Not it is not always empty. It is only empty by default, but, in my case, it is not empty when I am calling it from the place where the placeholder gets wiped. I am passing a folder name there.

Reply:
> > @allexzander So if I understand the PROPFIND that are called before deciding if it is a rename are making it decide something else ?
> 
> @mgallien
> About this, I am not sure. Where is PROPFIND called?

so maybe we can conclude this one and then we try to fix logic of rename and conflict detection because from the log I am not very happy with current state